packages/xo-server-usage-report/src/index.js

```diff
@@ -405,16 +405,26 @@ async function getSrsStats({ xo, xoObjects }) {
 }
 
 function computeGlobalVmsStats({ haltedVms, vmsStats, xo }) {
-  const allVms = concat(
-    map(vmsStats, vm => ({
-      uuid: vm.uuid,
-      name: vm.name,
-    })),
-    map(haltedVms, vm => ({
-      uuid: vm.uuid,
-      name: vm.name_label,
-    }))
-  )
+  const allVms = vmsStats.map(vm => ({
+    uuid: vm.uuid,
+    name: vm.name,
+  }))
+
+  haltedVms.forEach(vm => {
+    const isReplication =
+      'start' in vm.blockedOperations &&
+      vm.tags.some(
+        tag => tag === 'Disaster Recovery' || tag === 'Continuous Replication'
+      )
+
+    // Exclude replicated VMs from the VMs evolution because they can be added then deleted by the backup due to its retention
```

Comment:
It's unclear (it's unrelated to the retention), this one might be better:

```suggestion
    // Exclude replicated VMs because they keep being created/destroyed due to the implementation
```